corehq/ex-submodules/couchforms/models.py

```diff
@@ -226,20 +226,20 @@ def metadata(self):
 
     @property
     def time_start(self):
-        # Will be addressed in https://github.com/dimagi/commcare-hq/pull/19391/
-        return None
+        return self.metadata.timeStart
 
     @property
     def time_end(self):
-        return None
+        return self.metadata.timeEnd
 
     @property
     def commcare_version(self):
-        return None
+        return str(self.metadata.commcare_version)
```

Comment:
Does it matter that this returns bytes in python 2, but unicode in python 3?

Reply:
Here it shouldn't.